spec/support/factory_girl.rb

```diff
@@ -7,4 +7,12 @@
     password_confirmation 'password'
   end
 
+  factory :show do
+    sequence(:title) {|n| "show #{n}"}
```

Comment:
Use 2 (not 0) spaces for indentation.<br>Space between { and | missing.<br>Space missing inside }.